src/coreclr/vm/interoplibinterface_shared.cpp

```diff
@@ -154,3 +154,142 @@ void Interop::OnAfterGCScanRoots(_In_ bool isConcurrent)
         ObjCMarshalNative::AfterRefCountedHandleCallbacks();
 #endif // FEATURE_OBJCMARSHAL
 }
+
+#ifdef FEATURE_GCBRIDGE
+
+namespace
+{
+    Volatile<BOOL> g_GCBridgeActive = FALSE;
+    CLREvent* g_bridgeFinished = nullptr;
+
+    void ReleaseGCBridgeArgumentsWorker(
+        _In_ size_t sccsLen,
+        _In_ StronglyConnectedComponent* sccs,
+        _In_ ComponentCrossReference* ccrs)
+    {
+        CONTRACTL
+        {
+            NOTHROW;
+            GC_NOTRIGGER;
+        }
+        CONTRACTL_END;
+
+        // Memory was allocated for the collections by the GC.
+        // See callers of GCToEEInterface::TriggerGCBridge().
+
+        // Free memory in each of the SCCs
+        for (size_t i = 0; i < sccsLen; i++)
+        {
+            free(sccs[i].Context);
+        }
+        free(sccs);
+        free(ccrs);
+    }
+}
+
+bool Interop::IsGCBridgeActive()
+{
+    CONTRACTL
+    {
+        MODE_COOPERATIVE;
+    }
+    CONTRACTL_END;
```

Comment:
```suggestion
    LIMITED_METHOD_CONTRACT;
```
This should fix the CI failure